.env.example

```
@@ -32,4 +32,10 @@ RECONCILIATION_CRON=*/15 * * * *
 # ── Soroban / Payments contract (proportional cancellation reads) ─────────────
 SOROBAN_RPC_URL=https://soroban-testnet.stellar.org
 PAYMENTS_CONTRACT_ID=C...
-SOROBAN_NETWORK_PASSPHRASE=Test SDF Network ; September 2015
\ No newline at end of file
+SOROBAN_NETWORK_PASSPHRASE="Test SDF Network ; September 2015"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SOROBAN_NETWORK_PASSPHRASE key should go before the SOROBAN_RPC_URL key

(UnorderedKey)